src/main/java/org/reactome/release/qa/check/CoV2InfectionPathwayEventCheck.java

```diff
@@ -0,0 +1,191 @@
+       for (GKInstance summation : (Collection<GKInstance>) cov2Event.getAttributeValuesList(ReactomeJavaConstants.summation)) {
+            GKInstance createdInst = (GKInstance) summation.getAttributeValue(ReactomeJavaConstants.created);
+            List<GKInstance> modifieds = summation.getAttributeValuesList(ReactomeJavaConstants.modified);
+            if (modifieds.size() > 0) {
+                // At time of writing (August 2020), nothing is reported from this clause, but it is a good check.
+                GKInstance mostRecentModifiedInst = modifieds.get(modifieds.size() - 1);
+                // Only check Year/Month/Day since the changes took place more than a day before Curators got to look at them in gk_central.
+                LocalDate createdDateTime = getDateTimeFromInstance(createdInst);
+                LocalDate modifiedDateTime = getDateTimeFromInstance(mostRecentModifiedInst);
+                if (createdDateTime.isAfter(modifiedDateTime)) {
+                    return false;
+                }
+            // Should be at least 1 modified instance in CoV-1-to-CoV-2 instances.
+            } else {
+                return false;
+            }
+       }
+        return true;
+    }
+
+    /**
+     * Parses out the Date from dateTime attribute of the incoming InstanceEdit instance type.
+     * This value is formatted as 'yyyy-MM-dd hh:mm:ss', and the 'date' portion (1st half) is all that is needed for this test.
+     * @param instanceEditInst - GKInstance, from either the 'created' or 'modified' attribute of an instance.
+     * @return - LocalDate, in the format 'yyyy-MM-dd'.
+     * @throws Exception, thrown by MySQLAdaptor.
+     */
+    private LocalDate getDateTimeFromInstance(GKInstance instanceEditInst) throws Exception {
+        return LocalDate.parse(instanceEditInst.getAttributeValue(ReactomeJavaConstants.dateTime).toString().split(" ")[0]);
+    }
+
+    /**
+     * Checks if Event has a 2020 literatureReference. This is because CoV-2 evidence was very recent when this was being evaluated.
+     * @param cov2Event - GKInstance, Event contained within 'SARS-CoV-2 Infection' Pathway.
+     * @return - boolean, true if at least 1 2020 literatureReference, false if not.
+     * @throws Exception, thrown by MySQLAdaptor.
+     */
+    private boolean hasRecentLiteratureReference(GKInstance cov2Event) throws Exception {
+
+        boolean has2020LiteratureReference = false;
+        // Iterature through all literatureReference instances, checking the 'year' attribute.
+        for (GKInstance literatureReference : (Collection<GKInstance>) cov2Event.getAttributeValuesList(ReactomeJavaConstants.literatureReference)) {
+            if (literatureReference.getSchemClass().isa(ReactomeJavaConstants.LiteratureReference)) {
+                if (literatureReference.getAttributeValue(ReactomeJavaConstants.year).toString().equals("2020")) {
+                    has2020LiteratureReference = true;
+                    break;
```

Comment:
I still think that this is not a good usage of `break`. When I see a for-loop, I expect it to run a fixed number of times (for all the items that `getAttributeValuesList` returns, in this case). Here, break is used to escape the loop. There are two of them nested within multiple levels of conditional statements. I think this is a potential future hazard area, in terms of readability and understandability.

Reply:
I was implementing the suggestion here: https://github.com/reactome/release-qa/pull/26#discussion_r519002804, but I realize now you specified doing it as a `while` loop

Reply:
I didn't mean to _literally_ use `break` (it's very rare that I would actually suggest such a thing). I meant to break the while-loop's condition by invalidating it. Sorry for any confusion!

Reply:
I removed them. I kept the for loop, but added an 'if' check for the status of `has2020LiteratureReference` so that additional operations aren't done on it. If requested, I'll add in a while loop if that would be preferred

Reply:
I'll approve it as it is, but in the future, I'd recommend using a while-loop for situations where you don't know how many times you want the loop to run, or if you might want to end the loop early, instead of a for-loop with breaks/returns hidden within.